vite.config.ts

```diff
@@ -30,6 +30,8 @@ export default defineConfig({
         "src/db.ts",
         "src/server.ts",
         "src/**/index.ts",
+        "src/utilities/sample_data/*",
+        "src/utilities/loadSampleData.ts",
```

Comment:
This is a CLI utility, and testing for it is redundant.

Reply:
@anshgoyalevil 
It's not the CLI utility we should be testing but rather the logic of the functions that is included in the same. 
Like for example, is the `formatDatabase` function actually resetting the database or not, do the files for all the models that we are using to populate the database exist, etc.

The goal shouldn't be code coverage, but rather making sure that if in future if we add support for more models, to decide to rework the implementation of these functions, the same does not break the existing logic.

Reply:
@EshaanAgg 

Your concerns are valid, but take the example of Server.ts, are we testing it? No, because some functions and features shouldn't be tested. For example, we are creating an apollo server in server.ts, but would you go and test if the function is actually being returning a apollo server.

Similarly, we are working with FS in the loadSampleData.ts file.

We would need to mock fs, and yargs for the tests to work. We are acceesing the external functions which cannot be tested. 

I would write the test for it, but, I don't think it would help in any way.